docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "2.4"
+version: "3.4"
```

Comment:
This is a big change. I'm not sure we should be switching to v3 without a thorough audit of the effects. Afaik, v2 is still supported with Compose v2 so I would prefer to not change this.

Reply:
The comment in the same file links to documentation for v3 of compose files, so I decided to try fixing this. The JetBrains IDEs usually complain about some parts of this file (mainly, the extension fields), but when you update the version to 3.4, the red disappears. However, I re-checked and _extension fields_ were introduced in version 2.4[^1], so it's confusing.

I'll revert this change.
[^1]: https://docs.docker.com/compose/compose-file/compose-versioning/#version-24

Reply:
IIRC there was some issues with the resource limits when using version 3.x. But we can eventually switch up once we verify things are working normally. Can you create a an issue to do the audit and upgrade?

Reply:
Done: #4020